### PR TITLE
docs: remove docs - levski

### PR DESCRIPTION
**Pull request by ChristianDarr-personal**
**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [x] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?) - N/A
- [ ] I have fully tested (add details below) this the new feature or bug fix (if not, why?) - N/A
- [x] I have opened a PR for the related docs change (if not, why?)
  <link to docs PR>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,32 +34,7 @@ For a full walkthrough of building custom images, use this [guide.](./doc/guides
 A brief video demonstration of building and using the device service can be found [here.](https://www.youtube.com/watch?v=vZqd3j2Zn2Y)
 
 # Learn More 
-
```
Comment (lenny-goodell) [inline README.md:37]: Shouldn't the above sections bee remove also and just reference the Docs. Guide links above are broken.

**Review by lenny-goodell — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -24,42 +24,8 @@ The figure below illustrates the software flow through the architecture componen
 5. **Publish Inference Events/Data:** Analytics inferences are formatted and passed to the destination message bus specified in the request.
 6.  **Export Data:** Publish prepared (transformed, enriched, filtered, etc.) and groomed (formatted, compressed, encrypted, etc.) data to external systems (be it analytics package, enterprise or on-premises application, cloud systems like Azure IoT, AWS IoT, or Google IoT Core, etc.
 
-
```
Comment (lenny-goodell) [inline README.md:27]: Isn't the above Overview and How it works in the edgex-doc documentation? If not it should be and removed for here.

File: README.md [hunk 3]
```diff
@@ -24,42 +24,8 @@ The figure below illustrates the software flow through the architecture componen
...
-### Miscellaneous
-[Postman](./doc/test-with-postman.md)  
-[User Authentication](./doc/onvif-user-authentication.md)  
-
-## Resources
-[Learn more about EdgeX Core Metadata](https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-metadata/2.1.0)  
-[Learn more about EdgeX Core Command](https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-command/2.1.0)
-
+Visit the [EdgeX documentation page](https://docs.edgexfoundry.org/2.3/microservices/device/supported/device-onvif-camera/general) for more details.
```
Comment (lenny-goodell) [inline README.md:7]: The docs have a special version called `latest` which will always navigate to that latest release.

```suggestion
Visit the [EdgeX documentation page](https://docs.edgexfoundry.org/latest/microservices/device/supported/device-onvif-camera/general) for more details.
```

File: README.md [hunk 4]
```diff
@@ -24,42 +24,8 @@ The figure below illustrates the software flow through the architecture componen
...
 
 ## References
```
Comment (lenny-goodell) [inline README.md]: Can these references also be move to the edgex-docs and removed here?

IMO, this README should not have any content and redirect people to the edgex-docs which has all the content

**Review by lenny-goodell — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (ChristianDarr-personal) [reply README.md:27]: It is - I had just thought some basic info left on this page might be nice to leave, but I can remove it.

File: README.md (hunk 3 above)
Comment (ChristianDarr-personal) [reply README.md:7]: This pr is for the levski release. I renamed it for clarity.

Comment (lenny-goodell) [reply README.md:7]: Ok.

File: README.md (hunk 2 above)
Comment (lenny-goodell) [reply README.md:27]: Having it in two places creates a maintenance issue, thus why I prefer it be removed.

Comment (ChristianDarr-personal) [reply README.md:27]: Sounds good

**Review by lenny-goodell — APPROVED** [review]

**Review by presatish — APPROVED** [review]

**Pull request merged**